### PR TITLE
Admin dashboard orders

### LINKED ISSUE #43: User Story 32, Admin can see all orders
```
As an admin user
When I log into my dashboard, "/admin/dashboard"
Then I see all orders in the system.
For each order I see the following information:

- user who placed the order, which links to admin view of user profile
- order id
- date the order was created

Orders are sorted by "status" in this order:

- packaged
- pending
- shipped
- cancelled
```

DO THESE STORIES FIRST:
- User Story 56, Admin User Profile Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #82

depends on #14

### PR DESCRIPTION
**Pull request by renecasco**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - 98.41%

# What functionality does this accomplish?
Admin Dashboard Orders

# What did you struggle on to complete?
Sorting the orders and making the sort test work

# Implements/Fixes:
* description
closes #43 

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please include an emoji of how you feel about this branch:
👍

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -15,6 +15,10 @@ def total_quantity
     self.order_items.sum(:quantity)
   end
 
+  def self.sort_by_status
```
Comment (milevy1) [inline app/models/order.rb]: Without a model test, our spec/models test coverage will drop for this but that's okay.

Comment (renecasco) [reply app/models/order.rb]: Yeah, I thought it might redundant, especially given our time constraint. I tested the sorting with css on the feature test. Model coverage is at 98.41% .
